src/plugins/ImportKeras/ImportKeras.js

```diff
@@ -0,0 +1,307 @@
+
+            }
+        });
+        let layerName = this.core.getAttribute(layerNode, 'name');
+        this.layerInfo[layerName] = layerNode;
+
+    };
+
+    ImportKeras.prototype._addConfigurableNodes = function (layerNode, layerConfig) {
+        let allPointerNames = this.core.getValidPointerNames(layerNode);
+        let config = layerConfig.config;
+        if (allPointerNames.length > 0) {
+            this.logger.debug(`Layer ${this.core.getAttribute(layerNode, 'name')}` +
+                ` has following configurable attributes ${allPointerNames.join(", ")}`);
+            allPointerNames.forEach((pointer) => {
+                // This case, there is no extra config for this attribute
+
+                if (config[pointer] && typeof config[pointer] === 'string') {
+                    let configurableNode = this.core.createNode({
+                        parent: layerNode,
+                        base: this.META[config[pointer]]
+                    });
+                    // This will set the necessary pointers.
+                    this.core.setPointer(layerNode, pointer, configurableNode);
+                    this.logger.debug(`Added ${this.core.getAttribute(configurableNode, 'name')}`
+                        + ` as ${pointer} to the layer `
+                        + `${this.core.getAttribute(layerNode, 'name')}`);
+                }
+                // The configuration will exist but inside the object.
+                else {
+                    if (!config[pointer]) {
+                    } else {
+                        let pluggableNode = this.core.createNode({
+                            parent: layerNode,
+                            base: this.META[config[pointer].class_name]
+                        });
+                        this.logger.debug(`Added ${this.core.getAttribute(pluggableNode, 'name')} as ${pointer} to the layer ` +
+                            `${this.core.getAttribute(layerNode, 'name')}`);
+                        let validArgumentsForThisNode = this.core.getValidAttributeNames(pluggableNode);
+                        let configForAddedNode = config[pointer].config;
+                        if (validArgumentsForThisNode && configForAddedNode) {
+                            validArgumentsForThisNode.forEach((arg) => {
+                                if (configForAddedNode[arg]) {
+                                    this.core.setAttribute(pluggableNode, arg, this._Stringify(configForAddedNode[arg]));
+                                }
+                            });
+                        }
+                    }
+                }
+            });
+        }
+    };
+
+    // This method is used to convert javascript arrays to a tuple/ list in string Representation.
+    ImportKeras.prototype._Stringify = function (obj) {
+        if (obj instanceof Array) {
+            return '[' + obj.map((val) => {
+                if (val) {
+                    return val;

```

Comment:
I don't think this will work with 2D lists. Should probably be a recursive call here

Reply:
https://github.com/deepforge-dev/deepforge-keras/pull/119#discussion_r336241652. Addresses this